tests/integration/questionnaire/test_questionnaire_section_summary.py

```diff
@@ -0,0 +1,23 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestQuestionnaireCalculatedSummary(IntegrationTestCase):
+    def test_calculated_summary_headers_with_change_link(self):
+        self.launchSurvey("test_section_summary")
+        self.post()
+        self.post()
+        self.post()
+        self.post()
+        self.assertInBody("<th>Question</th>")
+        self.assertInBody("<th>Answer given</th>")
+        self.assertInBody("<th>Change answer</th>")
+
+    def test_calculated_summary_headers_without_change_link(self):
```

Comment:
These shouldn't be in a new file, we already have tests for summary (TestQuestionnaireSubmitWithSummary) and view submitted response (TestViewSubmissionResponse), it should live there.

Reply:
The name of the tests need changing, this is not doing anything from the calculated summary.